easybuild/easyconfigs/m/mrsFAST/mrsFAST-2.6.0.4-goolf-1.4.10.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'MakeCp'
+
+name = 'mrsFAST'
+version = '2.6.0.4'
+
+homepage = 'http://mrsfast.sourceforge.net'
+description = """ mrsFAST is designed to map short reads to reference genome assemblies 
+ in a fast and memory-efficient manner. mrsFAST is a cache-oblivous short read mapper that 
+ optimizes cache usage to get higher performance. """
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['https://github.com/sfu-compbio/mrsfast/archive/']
+sources = ['v%(version)s.tar.gz']
+
```

Comment:
@pescobar also apply the patch here, after making it generic (i.e. make it pick up `$CC`)